testdata/catalogs/test-catalog/catalog.yaml

```diff
@@ -0,0 +1,94 @@
+{
+    "schema": "olm.package",
+    "name": "prometheus",
+    "defaultChannel": "beta"
+}
+{
+    "schema": "olm.channel",
+    "name": "beta",
+    "package": "prometheus",
+    "entries": [
+        {
+            "name": "prometheusoperator.0.47.0"
+        }
+    ]
+}
+{
+    "schema": "olm.bundle",
+    "name": "prometheusoperator.0.47.0",
+    "package": "prometheus",
+    "image": "localhost/testdata/bundles/registry-v1/prometheus-operator:v0.47.0",
+    "properties": [
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "Alertmanager",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "AlertmanagerConfig",
+                "version": "v1alpha1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "PodMonitor",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "Probe",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "Prometheus",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "PrometheusRule",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "ServiceMonitor",
+                "version": "v1"
+            }
+        },
+        {
+            "type": "olm.gvk",
+            "value": {
+                "group": "monitoring.coreos.com",
+                "kind": "ThanosRuler",
+                "version": "v1"
+            }
+        },
```

Comment:
I'd kinda prefer that we add properties only as we decide we're going to use them and test something about them. That way we don't have a bunch of inert stuff in here that our future selves have to wonder if its necessary to keep.

Reply:
Makes total sense 👍🏽